driver-async/src/test/functional/com/mongodb/async/client/ChangeStreamsTest.java

```diff
@@ -118,10 +118,12 @@ public void setUp() {
             }
         }
 
+        CollectionHelper.dropDatabase(namespace.getDatabaseName(), WriteConcern.MAJORITY);
```

Comment:
Spec test README says to drop the databases.  It wasn't necessary until the rename test was added though.